### PR TITLE
Development

### PR DESCRIPTION
**Pull request by spicymaya**
Bad name for a branch. I'm sorry! I will name properly moving forward. All commit messages contain description of my changes.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,231 @@
+{
```
Comment (dialupdev) [inline package-lock.json]: Why do you have this file when you don't have a `package.json`?

File: .gitignore [hunk 2]
```diff
@@ -25,3 +25,5 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+./node-modules
```
Comment (celeryclub) [inline .gitignore]: I don't think you need this line.

**Review by dialupdev — APPROVED** [review]
Looks good! Left a couple of comments.

File: config.ru [hunk 3]
```diff
@@ -10,6 +10,6 @@ use Rack::Cors do
     origins '*'
     resource '*', 
         :headers => :any, 
-        :methods => [:get, :post, :delete, :put, :options]
+        :methods => [:get, :post, :delete, :put, :patch, :options]
```
Comment (dialupdev) [inline config.ru:13]: ⭐️

**Comment by dialupdev** [comment]
Also just a minor note that you can remove .DS_Store and add it to your gitignore. But this looks really really great!

File: package-lock.json (hunk 1 above)
Comment (spicymaya) [reply package-lock.json]: I have no idea. I removed it. The app runs fine, it doesn't seem to actually do anything.

**Comment by spicymaya** [comment]
I got all changes made and pushed development branch back up. You can merge it now.

**Pull request merged**